### ISSUE TITLE
When a suggestion spills to the next line, my cursor jumps to the beginning of that line

### ISSUE THREAD

**Issue by lagerspetz** [zsh-users/zsh-autosuggestions#226]
I am using zsh-autosuggestions, and with the git plugin, I have a long commit command in a narrow terminal in my history. when I type the beginning of it, e.g. git commit -m "D, then the long suggestion appears, spills to the next line, and my cursor appears at the beginning of the next line. I would expect the cursor to remain at "D_, not be placed on the new line.
Example:
/home/user/very/long/path/here/in/my/prompt/gitrepo> git commit -m "DY_epend_
X_encies added" build.sbt_

Where X is where the cursor jumps to, and Y is where it should be.

**Comment by ericfreese** [comment]
I'm not able to reproduce this. Can you give some more details? What version of this plugin? What OS? What version of zsh? etc.

Also, are you able to reproduce without loading your `.zshrc`. Try running `zsh -f` and sourcing `zsh-autosuggestions.zsh` manually. If the problem doesn't occur, that means it's something in your `.zshrc` causing the issue. It would then be very helpful for you to systematically pare down your `.zshrc` until the problem does not occur to nail down exactly what's causing it. A screenshot or screen recording would also be very helpful for visualizing the issue.

Thanks!

**Comment by lagerspetz** [comment]
I have investigated a bit further. This seems to occur with the oh-my-zsh awesomepanda theme, and has something to do with the 2-char tick mark or cross symbol that the theme shows for dirty/clean git repositories.
To reproduce:
Install oh my zsh http://ohmyz.sh/ and set ZSH_THEME="awesomepanda"
Install autosuggestions for oh my zsh, enable plugins git dirhistory zsh-autosuggestions

1. Type something
2. get autosuggestions show up
3. Type something different so it is cleared
4. See the git dirty/clean symbol disappear and command shifted to the left, some characters disappear
5. press home and type a hash, then press enter
6. Now press up and see how the command was different than what was shown

Various versions:
zsh 5.2 (x86_64-ubuntu-linux-gnu)
oh-my-zsh master commit b908feebcfb0ca8a9a80360d177e716c24c317d6
autosuggestions master commit fedc22e9bbd046867860e772d7d6787f5dae9d4c

**Comment by Hultner** [comment]
@lagerspetz Does your problem look like this 

https://asciinema.org/a/16v4abb56t9gq9nu7jwow4g5o 

That's from #153 which is a problem I'm experiencing as well, maybe it could help in narrowing it down.

**Comment by lagerspetz** [comment]
Yes that looks like what I am experiencing. I use oh-my-zsh and after changing to the "mgutz" theme, it happens much less, but it still does happen occasionally.

**Comment by c-dante** [comment]
Chiming in that I have the same problem -- seems the autocomplete sometimes jumps the cursor to a different position during display but does not change the buffer.
